test/functional/account_controller_test.rb

```diff
@@ -32,53 +32,51 @@
 # Re-raise errors caught by the controller.
 class AccountController; def rescue_action(e) raise e end; end
 
-class AccountControllerTest < ActionController::TestCase
+describe AccountController, type: :controller do
+  render_views
+
   fixtures :all
 
-  def setup
-    super
-    @controller = AccountController.new
-    @request    = ActionController::TestRequest.new
-    @response   = ActionController::TestResponse.new
+  before do
     User.current = nil
   end
 
-  def test_login_with_wrong_password
+  it 'should login with wrong password' do
     post :login, username: 'admin', password: 'bad'
     assert_response :success
     assert_template 'login'
     assert_select 'div.flash.error.icon.icon-error', /Invalid user or password/
   end
 
-  def test_login
+  it 'should login' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.